frontend/e-buddy/src/app/(demo)/bots/view/page.tsx

```
@@ -0,0 +1,233 @@
+                  {bot.createdAt}
+                </div>
+                <div className="col-span-1 sm:col-span-3 flex justify-end space-x-2">
+                  <button
+                    onClick={(e) => {
+                      e.stopPropagation();
+                      router.push(`/bots/${bot.id}`);
+                    }}
+                    title="Embed Widget"
+                    className="p-2 rounded hover:bg-gray-100"
+                  >
+                    <Code className="h-5 w-5 text-gray-600 hover:text-indigo-600" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)